src/components/combobox/combobox.tsx

```diff
@@ -628,6 +628,10 @@ export class Combobox
         }
         break;
       case "Escape":
+        if (!this.open) {
```

Comment:
should `this.clearDisabled` be checked here?

Reply:
Yes, it should. Sorry about that. 😞

Reply:
FIxed!